docs/book/src/topics/vm-diagnostics.md

```diff
@@ -0,0 +1,78 @@
+# VM Diagnostics
+
+This document describes how configure the VM Diagnostics options in Azure.
+
+The VM Diagnostics options allow you to configure options for troubleshoot the startup of an Azure Virtual Machine.
+You can Use this feature to troubleshoot boot failures for custom or platform images.
```

Comment:
```suggestion
You can use this feature to investigate boot failures for custom or platform images.
```
(We just said "troubleshoot" in the previous sentence, so I used "investigate" here for variety.)